flowforge/input/System.py

```diff
@@ -162,11 +173,22 @@ def _setupSimpleLoop(
         loop : List[dict]
             List specifying the construction of loop via component names and forces.  Ordering is
             from the 'first' component of the loop to the 'last'.
+        boundary_conditions : Dict
+            Dictionary of boundary conditions for the segment
+        fluid : str
+            The working fluid used in the segment (e.g., "FLiBe"). Defaults to "FLiBe".
+        gas  optional :
+            An optional parameter to specify gas in the system (e.g. "Helium")
+
         """
         self._isLoop = True
 
         components, loop = make_continuous(components, loop)
         self._fluidname = fluid.lower()
+        if gas is not None:
+            self._gasname = gas.lower()
+        else:
+            self._gasname = gas
```

Comment:
shorter to just say:

`self._gasname = gas if gas is None else gas.lower()`

Reply:
Agreed